documentation/operators/src/nodes/nym-api.md

```diff
@@ -0,0 +1,162 @@
+[//]: # ()
+
+> The `nym-api` binary should be coming out in the next release - we're releasing this document beforehand so that Validators have information as soon as possible and get an idea of what to expect. This doc will be expanded over time as we release the API binary itself as well as start enabling functionality.  
+
+## What is the Nym API?
+The Nym API is a binary that will be operated by some or all of the Nyx Blockchain Validator set. THie binary can be run in several different modes, and has two main bits of functionality: 
+* network monitoring (calculating the routing score of Mixnet nodes) 
+* generation and validation of [zk-Nyms](https://blog.nymtech.net/zk-nyms-are-here-a-major-milestone-towards-a-market-ready-mixnet-a3470c9ab10a), our implementation of the Coconut Selective Disclosure Credential Scheme.
+
+This is important for both the proper decentralisation of the network uptime calculation and, more pressingly, enabling the NymVPN to utilise privacy preserving payments. 
+
+The process of enabling these different aspects of the system will take time. Operators of the Nym API binary for the moment will only have to run the binary in a minimal 'caching' mode in order to get used to maintaining an additional process running alongside their Validator. 
+
+### Rewards 
+Operators of Nym API instances will be rewarded for performing the extra work of taking part in credential generation. These rewards will be calculated **separately** from rewards for block production. 
+
+Rewards for credential signing will be calculated hourly, with API operators receiving a proportional amount of the reward pool (333NYM per hour / 237600NYM per month) according to the % of credentials they have signed. 
+
+### (Coming Soon) Machine Specs 
+We are working on load testing currently in order to get good specs for a Validator + Nym API setup. Bear in mind that credential signing is primarily CPU-bound. 
+
+### (Coming Soon) Credential Generation
+Validators that take part in the DKG ceremony (more details on this soon) will become part of the quorum generating and verifying zk-Nym credentials. These will initially be used for private proof of payment for NymVPN (see our blogposts [here](https://blog.nymtech.net/nymvpn-an-invitation-for-privacy-experts-and-enthusiasts-63644139d09d) and [here](https://blog.nymtech.net/zk-nyms-are-here-a-major-milestone-towards-a-market-ready-mixnet-a3470c9ab10a) for more on this), and in the future will be expanded into more general usecases such as [offline ecash](https://arxiv.org/abs/2303.08221). 
+
+## Current version
+```
+<!-- cmdrun ../../../../target/release/nym-api --version | grep "Build Version" | cut -b 21-26  -->
+```
+
+## Setup and Usage
+### Viewing command help
+You can check that your binary is properly compiled with:
+
+```
+./nym-api --help
+```
+
+Which should return a list of all available commands.
+
+~~~admonish example collapsible=true title="Console output"
```

Comment:
I get this output:
```
Starting nym api...

```
I bet it catches whatever comes first and doesn't print what comes next to the output window.

Reply:
Hmm it shouldnt be starting it but init-ing it... let me check

Reply:
Ah so even when you run just `--help` it will log `starting`. Locally the entire message is being rendered by `cmdrun` will redeploy to staging and check again now